lite2/example_test.go

```diff
@@ -48,17 +52,16 @@ func TestExample_Client_AutoUpdate(t *testing.T) {
 	if err != nil {
 		stdlog.Fatal(err)
 	}
+	trustedStore := dbs.New(db, chainID)
+
+	err = trustedStore.SaveSignedHeaderAndValidatorSet(header, vals)
 
-	c, err := NewClient(
+	c, err := NewNetClientFromTrustedStore(
```

Comment:
better to create another test

Reply:
these new functions test the old ones as well so by writing a test for the new ones you are also still testing the old ones

Reply:
example tests should show how to use light client in "real life". we shouldn't expose store and presave headers there.